data/tlg0007/tlg082/__cts__.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ti:work xmlns:ti="http://chs.harvard.edu/xmlns/cts" groupUrn="urn:cts:greekLit:tlg0007" projid="greekLit:tlg082" urn="urn:cts:greekLit:tlg0007.tlg082" xml:lang="grc">
+    <ti:title xml:lang="lat">Apophthegmata Laconica</ti:title>
+    
+    <ti:translation urn="urn:cts:greekLit:tlg0007.tlg082.perseus-eng3" workUrn="urn:cts:greekLit:tlg0007.tlg082" xml:lang="eng">
+        <ti:label xml:lang="eng">Sayings of Spartans</ti:label>
+        <ti:description xml:lang="mul">Plutarch. Moralia, Vol. III. Babbitt, Frank Cole, translator. Cambridge, MA: Harvard University Press. London: William Heinemann Ltd. 1931.</ti:description>
+    </ti:translation>
+    
+    <ti:translation urn="urn:cts:greekLit:tlg0007.tlg082.perseus-eng4" workUrn="urn:cts:greekLit:tlg0007.tlg082" xml:lang="eng">
+        <ti:label xml:lang="eng">Laconic Apophthegms; or Remarkable Sayings of the Spartans.</ti:label>
+        <ti:description xml:lang="eng">Plutarch. Plutarch's Morals, Vol. I. Goodwin, William W., editor; unknown, translator. Boston: Little, Brown, and Company. Cambridge. 1874 (printing).</ti:description>
+    </ti:translation>
+    
+    <ti:edition urn="urn:cts:greekLit:tlg0007.tlg082.perseus-grc3" workUrn="urn:cts:greekLit:tlg0007.tlg082" xml:lang="grc">
+        <ti:label xml:lang="grc">Ἀποφθέγματα Λακωνικά</ti:label>
+        <ti:description xml:lang="mul">Plutarch. Moralia, Vol. III. Babbitt, Frank Cole, translator. Cambridge, MA: Harvard University Press. London: William Heinemann Ltd. 1931</ti:description>
+    </ti:edition>
+    
+    <ti:edition urn="urn:cts:greekLit:tlg0007.tlg082.perseus-grc4" workUrn="urn:cts:greekLit:tlg0007.tlg082" xml:lang="grc">
+        <ti:label xml:lang="grc">Ἀποφθέγματα Λακωνικά</ti:label>
+        <ti:description xml:lang="mul">Plutarch. Plutarchi Chaeronensis Moralia, Vol II. Bernardakis, Gregorius N., editor; Leipzig: Teubner. 1889.</ti:description>
```

Comment:
please revise the name to read as in the Perseus Catalog
http://catalog.perseus.org/catalog/urn:cts:greekLit:tlg0007.tlg082.perseus-grc2
`Vernardakēs, Grēgorios N.`